source/server/util.js

```diff
@@ -69,52 +71,106 @@ export function getCurrentRoundState(gameInstance) {
-      return 1;
+      if (suite2 === SUITES.CUPS) multiplier = worldEvent === WORLD_EVENTS.SUITE_BOOST_WANDS ? MULTIPLER.GREATER + MULTIPLER.BOOST : MULTIPLER.GREATER;
+      else if (suite2 == SUITES.SWORDS) multiplier = worldEvent === WORLD_EVENTS.SUITE_BOOST_WANDS ? MULTIPLER.LESS + MULTIPLER.BOOST : MULTIPLER.LESS;
     case SUITES.CUPS:
-      if (card2.suite === SUITES.SWORDS) return 2;
-      else if (card2.suite == SUITES.WANDS) return 0.5;
-      return 1;
+      if (suite2 === SUITES.SWORDS) multiplier =  worldEvent === WORLD_EVENTS.SUITE_BOOST_CUPS ? MULTIPLER.GREATER + MULTIPLER.BOOST : MULTIPLER.GREATER;
+      else if (suite2 == SUITES.WANDS) multiplier =  worldEvent === WORLD_EVENTS.SUITE_BOOST_CUPS ? MULTIPLER.LESS + MULTIPLER.BOOST : MULTIPLER.LESS;
     case SUITES.SWORDS:
-      if (card2.suite === SUITES.WANDS) return 2;
-      else if (card2.suite == SUITES.CUPS) return 0.5;
-      return 1;
+      if (suite2 === SUITES.WANDS) multiplier =  worldEvent === WORLD_EVENTS.SUITE_BOOST_SWORDS ? MULTIPLER.GREATER + MULTIPLER.BOOST : MULTIPLER.GREATER;
+      else if (suite2 == SUITES.CUPS) multiplier =  worldEvent === WORLD_EVENTS.SUITE_BOOST_SWORDS ? MULTIPLER.LESS + MULTIPLER.BOOST : MULTIPLER.LESS;
+    case SUITES.PENTACLES:
+      multiplier = worldEvent === WORLD_EVENTS.SUITE_BOOST_PENTACLES ? MULTIPLER.NEUTRAL + MULTIPLER.BOOST : MULTIPLER.NEUTRAL;
     default:
-      return 1;
+
   }
+
+  return worldEvent === WORLD_EVENTS.SUITE_REVERSED ? 1/multiplier : multiplier;
+
 } /* getMultiplier */
 
+function getRandomValue() {
+  return Math.floor(Math.random() * 14) + 1;
+}
+
 /**
  * Determines which of two cards wins a round
- * @param { Types.Card } card1 card to compare
- * @param { Types.Card } card2 card to compare
- * @returns { Types.Card } winning card of card1 and card2
+ * @param { Types.GameInstance } gameInstance
+ * @returns { Types.UUID } winning user UUID1 and UUID2
  */
-export function getWinningCard(card1, card2) {
-  return card1.number * getMultiplier(card1, card2) > card2.number
-    ? card1
-    : card2;
-} /* getWinningCard */
+
+export function getRoundWinnerUUID(gameInstance) {
+  const currentRoundState = getCurrentRoundState(gameInstance);
+  const worldEvent = currentRoundState.worldEvent;
+  const [[uuid1, card1], [uuid2, card2]] = Object.entries(
+    currentRoundState.selectedCard,
+  );
+
+  switch (worldEvent) {
+    case WORLD_EVENTS.LOWER_WINS:
+      return card1.number < card2.number ? uuid1 : uuid2;
+    case WORLD_EVENTS.RANDOM_VALUE:
+      return getRandomValue() * getMultiplierWorldEvent(card1, card2, worldEvent) > getRandomValue() ? uuid1 : uuid2;
+    /*fallthrough because other world events influence the multiplier method not winning card*/
+    default:
+      return card1.number * getMultiplierWorldEvent(card1, card2, worldEvent) > card2.number? uuid1: uuid2;
+  }
+}
+
+function getRandomWorldEvent(gameInstance){
+  const currentRound = gameInstance.gameState.byRound.length + 1;
```

Comment:
This should be named something like `currentRoundNumber`, b/c its not the current round itself